metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -383,6 +384,28 @@ class MetalsLanguageServer(
       buffers,
       definitionProvider
     )
+    superMethodProvider = new SuperMethodProvider(
+      definitionProvider,
+      implementationProvider.findSemanticDbWithPathForSymbol
+    )
+
+    goToSuperMethod = new GoToSuperMethod(
```

Comment:
Can we keep `SuperMethodProvider` and `GoToSuperMethod` together? It's not really a separate functionality.